Stripe/PublicHeaders/STPPaymentHandler.h

```diff
@@ -159,7 +159,7 @@ withAuthenticationContext:(id<STPAuthenticationContext>)authenticationContext
  
  @see https://stripe.com/docs/payments/cards/saving-cards#saving-card-without-payment
  
- @param setupIntentConfirmParams The params used to confirm the SetupIntent.
+ @param setupIntentConfirmParams The params used to confirm the SetupIntent. Note that use of STPPaymentHandler enforces that a useStripeSDK value of true is set during the confirm call, regardless of setupIntentConfirmParams.useStripeSDK value.
```

Comment:
brevity nit, feel free to take or leave: "Note that this method overrides the value of paymentParams.useStripeSDK to `YES`".